partner_disable_gravatar/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Partner Disable Gravatar
+========================
+
+A new feature has been introduced in Odoo v10 in the *base* module: when you enter an email address in the form view of a partner, Odoo will send a query to `gravatar.com <https://www.gravatar.com/>`_ to get a picture corresponding to the email address.
```

Comment:
I think you can drop this paragraph, since its idea is repeated below in a better way.